### PR TITLE
Fix pennyworth import

### PR DESCRIPTION
**Pull request by thardeck**

### CODE REVIEW

File: lib/pennyworth/commands/import_base_command.rb [hunk 1]
```diff
@@ -84,6 +84,18 @@ def read_remote_box_state_file
       end
     end
 
+    def get_virsh_image_name(box)
```
Comment (aduffeck) [inline lib/pennyworth/commands/import_base_command.rb]: Shouldn't this rather be `virsh_image_name`?

File: lib/pennyworth/commands/import_base_command.rb [hunk 2]
```diff
@@ -84,6 +84,18 @@ def read_remote_box_state_file
...
+      output = Cheetah.run(
+        "virsh", "-c", "qemu:///system", "vol-list", "--pool=default", stdout: :capture
+      )
+      result = output.scan(/^ (#{box}\S*)/).first
```
Comment (aduffeck) [inline lib/pennyworth/commands/import_base_command.rb]: I'd replace lines 91-97 with `output[/^ (#{box}\S*)/, 1]`

File: lib/pennyworth/commands/import_base_command.rb [hunk 3]
```diff
@@ -92,10 +104,14 @@ def fetch_remote_box_state_file
...
-      @vagrant.destroy
-    rescue
+      image = get_virsh_image_name(box)
+      if !image.empty?
+        Cheetah.run(
+          "virsh", "-c", "qemu:///system", "vol-delete", image, "--pool=default"
+        )
+        Cheetah.run "virsh", "-c", "qemu:///system", "pool-refresh", "--pool=default"
+        @vagrant.destroy
+      end
```
Comment (aduffeck) [inline lib/pennyworth/commands/import_base_command.rb]: Should we raise an error or a warning if there is no corresponding image?

Comment (thardeck) [inline lib/pennyworth/commands/import_base_command.rb]: No, since import images should work without errors if there was no image imported before.

**Comment by aduffeck** [comment]
lgtm

**Pull request merged**